src/client/pages/store-management/store-management-view.html

```diff
@@ -0,0 +1,32 @@
+<div class="store-management-view">
+    <div class="container is-fullhd">
+        <section>
+            <div class="hero is-primary">
+                <div class="title is-1">Store Management</div>
+            </div>
+            <div class="columns is-grouped">
+                <div class="hero lined">
+                    <div class="navbar1">
+                        <router-link to="/invoicePage">
+                            <a class="navbar-item" v-on:click.prevent="createInvoice">Create new invoice</a>
+                        </router-link>
+                        <a class="navbar-item" :class="{'is-active':selectedNav === 'viewTotalSales'}"
```

Comment:
you can use selectedTab here so that you need to set selectedTab only in view model as well.